kalite/main/api_views.py

```diff
@@ -34,6 +34,9 @@ def search_api(request, channel):
 
     matches, exact, pages = search_topic_nodes(query=query, channel=channel, language=request.language, page=1, items_per_page=15, exact=False)
 
+    if matches==list():
```

Comment:
A more Pythonic way to do this would be to test matches for emptiness, rather than seeing if it equals an empty list.

In Python empty objects are False-y, so we could just do:
```
if not matches:
```